src/initializers/sequelize.ts

```diff
@@ -96,4 +107,15 @@ export class SequelizeInitializer extends Initializer {
 
     await api.sequelize.query(query);
   }
+
+  injectSchema(original, schema) {
```

Comment:
I think this method should check which `dialect` we are using - I /think/ that this schema work only applies to postgres?